components/contribute-cards/ContributeTier.js

```diff
@@ -50,7 +55,8 @@ const ContributeTier = ({ intl, collective, tier, ...props }) => {
   const currency = tier.currency || collective.currency;
   const isFlexibleAmount = tier.amountType === 'FLEXIBLE';
   const minAmount = isFlexibleAmount ? tier.minimumAmount : tier.amount;
-  const amountRaised = tier.interval ? tier.stats.totalRecurringDonations : tier.stats.totalDonated;
+  const amountRaised =
+    tier?.interval !== INTERVALS.flexible ? tier.stats.totalRecurringDonations : tier.stats.totalDonated;
```

Comment:
This condition doesn't work, if `tier.interval` is `null` then `tier?.iterval !== INTERVALS.flexible` will return true